apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Types/Balance/BalanceResponsibilityMeteringPointType.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Energinet.DataHub.WebApi.GraphQL.Extensions;
+using ESett = Energinet.DataHub.WebApi.Clients.ESettExchange.v1;
+
+namespace Energinet.DataHub.WebApi.GraphQL.Types.Balance;
+
+public class BalanceResponsibilityMeteringPointType : EnumType<ESett.MeteringPointType>
+{
+    protected override void Configure(IEnumTypeDescriptor<ESett.MeteringPointType> descriptor)
+    {
+        descriptor.Name("BalanceResponsibilityMeteringPointType");
+        descriptor.AsIsCase();
+    }
+}
```

Comment:
issue: looks like this causes an issue with the metering point type translations in the overview table. The value is not translated anymore.

![image](https://github.com/user-attachments/assets/07a9e9d6-ace5-4521-a760-5bba0e6b1e22)

Reply:
Same translation was used for both timeseriestype and metering point time. Fixed now